core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -1084,7 +1080,9 @@ private void deleteAndJournal(LockedInodePath inodePath, DeleteOptions deleteOpt
     long opTimeMs = System.currentTimeMillis();
     deleteInternal(inodePath, false, opTimeMs, deleteOptions);
     DeleteFileEntry deleteFile = DeleteFileEntry.newBuilder().setId(fileId)
-        .setRecursive(deleteOptions.isRecursive()).setOpTimeMs(opTimeMs).build();
```

Comment:
I updated `alluxio/master/journal/AbstractJournalFormatterTest.java`. In terms of unit testing this particular logic, we don't have a good way to do so as the `fooAndJournal` methods are private. 
And the `FileSystemMasterTest` currently do have this facilities too.